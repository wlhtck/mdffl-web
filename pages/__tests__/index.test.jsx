import React from 'react';
import { mount } from 'enzyme';
import Index from '../index';

describe('pages/index', () => {
  it('renders an index page', () => {
    expect(mount(<Index router={{ query: { slug: 'index' } }} />)).toMatchSnapshot();
  });
});
