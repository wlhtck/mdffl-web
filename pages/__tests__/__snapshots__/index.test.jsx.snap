// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index renders an index page 1`] = `
<withRouter(Unknown)
  router={
    Object {
      "query": Object {
        "slug": "index",
      },
    }
  }
>
  <Component
    router={
      Object {
        "query": Object {
          "slug": "index",
        },
      }
    }
  >
    <heroBlock>
      <div />
    </heroBlock>
    <ContentBlock
      body="The bear is super cool and everyone loves him! Click below to find out more!"
      className=""
      cta={
        Object {
          "external": true,
          "text": "Google",
          "url": "https://www.google.com/search?q=Bear",
        }
      }
      headline="Check out the bear!"
      image={
        Object {
          "alt": "Bear",
          "src": "//images.ctfassets.net/8wpi5cmhr3lg/6mW6Tsx5scOYMwocwSskKq/f1bcb0a3f979f7de3e1ba070d0d14f4f/photo-1541560584704-f2767960951a.jpeg",
        }
      }
      imageAlign="right"
      slug="bear"
      textAlign="left"
      type="contentBlock"
    >
      <Row
        padding={null}
        reverse={true}
      >
        <div
          className="Row-sc-9u807i-0 bAatiY"
        >
          <Col
            align="center"
            padding={null}
            reverse={false}
          >
            <div
              className="Col-inu58n-0 jGVFOM"
            >
              <Image
                alt="Bear"
                src="//images.ctfassets.net/8wpi5cmhr3lg/6mW6Tsx5scOYMwocwSskKq/f1bcb0a3f979f7de3e1ba070d0d14f4f/photo-1541560584704-f2767960951a.jpeg"
              >
                <Image
                  alt="Bear"
                  className="Image-sc-1mf1fvy-0 dHEHxo"
                  src="//images.ctfassets.net/8wpi5cmhr3lg/6mW6Tsx5scOYMwocwSskKq/f1bcb0a3f979f7de3e1ba070d0d14f4f/photo-1541560584704-f2767960951a.jpeg"
                >
                  <img
                    alt="Bear"
                    className="Image-sc-1mf1fvy-0 dHEHxo"
                    src="//images.ctfassets.net/8wpi5cmhr3lg/6mW6Tsx5scOYMwocwSskKq/f1bcb0a3f979f7de3e1ba070d0d14f4f/photo-1541560584704-f2767960951a.jpeg"
                  />
                </Image>
              </Image>
            </div>
          </Col>
          <Col
            align="flex-start"
            padding={65}
            reverse={false}
          >
            <div
              className="Col-inu58n-0 loWYpg"
            >
              <Typography__FeatureHeadline>
                <h2
                  className="Typography__FeatureHeadline-sc-1h5b1wa-1 ijDGCs"
                >
                  Check out the bear!
                </h2>
              </Typography__FeatureHeadline>
              <Typography__BodyCopy
                inverted={false}
                textAlign="left"
              >
                <p
                  className="Typography__BodyCopy-sc-1h5b1wa-2 cCAEZM"
                >
                  The bear is super cool and everyone loves him! Click below to find out more!
                </p>
              </Typography__BodyCopy>
              <Button
                external={true}
                text="Google"
                type="primary"
                url="https://www.google.com/search?q=Bear"
              >
                <Button
                  className="Button-sc-19aup1o-0 kdsCVw"
                  external={true}
                  onClick={null}
                  text="Google"
                  type="primary"
                  url="https://www.google.com/search?q=Bear"
                >
                  <a
                    className="Button-sc-19aup1o-0 kdsCVw"
                    href="https://www.google.com/search?q=Bear"
                    rel="noreferrer noopener"
                    target="_blank"
                  >
                    Google
                  </a>
                </Button>
              </Button>
            </div>
          </Col>
        </div>
      </Row>
    </ContentBlock>
  </Component>
</withRouter(Unknown)>
`;
